code/outlierfunction.py

```diff
@@ -0,0 +1,16 @@
+
+# find outliers based on DVARS and FD
+
+def outlier(file_name, bound):
```

Comment:
Can we make change outlier take in arrays instead of files? load the txt file in the script. This way we dont have to create a text file every time we run the test. In general, try to make function parameters be data types, not files;  thanks